packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -390,6 +390,877 @@ export class CustomRule extends RuleNew {
   }
 }
 
+/**
+ * Managed rules that are supported by AWS Config.
+ * @see https://docs.aws.amazon.com/config/latest/developerguide/managed-rules-by-aws-config.html
+ */
+export class ManagedRuleIdentifier {
```

Comment:
Now that we drifted away from the enum-like class here... Does it make sense to name this `ManagedRuleIdentifiers`?

Reply:
good call out. that seems sensible with this shift. updating!

Reply:
done.